src/app/listener.ts

```diff
@@ -69,6 +80,17 @@ export class Listener {
       notificationChannel.send({ embeds: [embed] });
     });
 
+    // Used to handle slash commands.
+    this.container.clientService.on('interactionCreate', (interaction: Interaction) => {
+      // If it's not a command, we don't care.
+      if (!interaction.isCommand()) {
```

Comment:
Commands can be user or message context menu commands, which isn't supported in lion yet. So we should specifically check against those cases just to be safe.
```suggestion
      if (!interaction.isChatInputCommandInteraction()) {
```

Reply:
This doesn't exist in Discord.js v13.8 which we're on currently. Not sure if we want to roll an update in this PR or not

Reply:
Oh yeah I forgot this is still on v13. We can keep this as-is as it's logically equivalent to v14's `isChatInputCommandInteraction()`.